.github/workflows/gobump.yml

```diff
@@ -15,8 +15,8 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Run gobump-deps action
-        uses: lzap/gobump@v1
+        uses: lzap/gobump@main
```

Comment:
I thought GHA uses semantic versioning and will pickup the latest v1.0.9 but it does not work that way. I would need to introduce `v1` branch, which is not worth the effort for such a simple GHA I think.